src/scenes/Portfolio/components/Works/components/List/List.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import Img from 'gatsby-image';
+
+import { useDribbbleShots } from '@hooks';
+
+import styles from './List.module.scss';
+
+const List = () => {
+  const items = useDribbbleShots();
+
+  const stepValue = 8;
+  const [numberOfRendered, setNumberOfRendered] = useState(stepValue);
+  const handleSetNumberOfRendered = () => {
+    setNumberOfRendered(numberOfRendered + stepValue);
+  };
+
+  return (
+    <div className={styles.container}>
+      <ul className={styles.list}>
+        {items.map((item, index) => {
+          return numberOfRendered > index ? (
+            <li key={index} className={styles.item}>
+              <a
+                href={item.url}
+                title={item.title}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <Img fluid={item.localCover.childImageSharp.fluid} />
+              </a>
+            </li>
+          ) : null;
+        })}
+      </ul>
+      <div className={styles.buttonWrapper}>
+        {numberOfRendered >= items.length ? (
+          <a
+            href="https://dribbble.com/halolab"
+            target="_blank"
+            rel="noopener noreferrer"
+            className={styles.button}
+          >
+            SEE ALL
+            <svg
```

Comment:
Didn't you want to use some react-svg module?

Reply:
I try but plugin description says: 
`Note: the plugin can remove SVGs from the built-in url-loader config in case invalid configuration.`
... and it destroy everything. At the moment this issue will take unreasonably long time. I am thinking about the solution.